### PR TITLE
Scrape ca_on_toronto committees

### PR DESCRIPTION
**Pull request by patcon**
Pending PR

### CODE REVIEW

**Comment by patcon** [comment]
I think this is good for review now:

- now scraping all committees for sessions available on tmmis (adding alt names as necessary based on shared two-letter committee codes)
- decision body ids stored by term in extras
- description of most recent term held in extras
- jurisdiction scraper overridden to add "City Council" alt name
- bills and events should now be resolving orgs for related_entities and participants
- helpers, abstraction, constants

File: ca_on_toronto/helpers.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import re
+from urllib.parse import parse_qs, urlparse
+
+from .constants import (
+    COMMITTEE_LIST_TEMPLATE,
+    )
+
+class Helpers():
```
Comment (jpmckinney) [inline ca_on_toronto/helpers.py]: Why wrap these in a class?

File: ca_on_toronto/jurisdiction.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from .constants import (
+    TWO_LETTER_ORG_CODE_SCHEME,
+    )
+
+class TorontoJurisdiction(CanadianJurisdiction):
+    def __init__(self):
+        super(TorontoJurisdiction, self).__init__()
+
+    def get_organizations(self):
+        for org in super(TorontoJurisdiction, self).get_organizations():
```
Comment (jpmckinney) [inline ca_on_toronto/jurisdiction.py]: If the supermethod ever yields other organizations, you won't want to make them all "City Council".

**Comment by patcon** [comment]
Comments addressed :)

**Comment by jpmckinney** [comment]
Cool, ready to squash!

**Comment by patcon** [comment]
Wooooo. Done and done.

**Pull request merged**